tests/test_ckan_backend.py

```diff
@@ -179,7 +179,7 @@ def all_metadata(resource_data):
         'name': faker.unique_string(),
         'title': faker.sentence(),
         'notes': faker.paragraph(),
-        'tags': [{'name': faker.unique_string()} for _ in range(3)],
+        'tags': [{'name': faker.unique_string(), 'id': faker.unique_string()} for _ in range(3)],
```

Comment:
I don't understand the change that would explain *why* it wasn't failing earlier (like last week on master)

Reply:
This seems to be due to deps cache in CI, since increasing the cache key on this branch [makes test fail as well](https://app.circleci.com/pipelines/github/opendatateam/udata-ckan?branch=feat%2Fa-fork)